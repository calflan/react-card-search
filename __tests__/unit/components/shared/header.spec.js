import React from 'react';
import Header from '../../../../src/js/components/shared/header';

describe('Contact Component', () => {
  let component;

  beforeEach(() => {
    component = shallow(<Header />);
  });

  test('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});