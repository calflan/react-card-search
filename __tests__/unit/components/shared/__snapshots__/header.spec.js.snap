// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component should match snapshot 1`] = `
<header>
  <img
    alt="Moonpig logo"
    id="logo"
    role="img"
    src="https://i.imgur.com/RpDClsG.png"
  />
</header>
`;
